src/main/java/com/pusher/client/channel/PusherEvent.java

```diff
@@ -80,12 +80,23 @@ public String toJson() {
     }
 
     public static PusherEvent fromJson(String json) {
```

Comment:
How about adding tests to it? Is there a way to reproduce the problem in a test case?